src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestSwitchCase4.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Please duplicate this input, but use https://checkstyle.sourceforge.io/property_types.html#RightCurlyOption:~:text=return%20id%3B%20%7D-,same,-Works%20like%20alone_or_singleline. According to https://checkstyle.sourceforge.io/property_types.html#RightCurlyOption:~:text=Works%20like%20alone_or_singleline%20but%20the%20brace%20should%20be%20on%20the%20same%20line%20as%20the%20next%20part%20of%20a%20multi%2Dblock%20statement, behavior should be exactly the same as this input.

Reply:
done